tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+envlist = py36,py37,py38,py39,lint
+
+[testenv]
+extras = test
+commands = py.test {posargs}
+
+[testenv:lint]
+basepython = python3.9
+skip_install=true
+deps =
+    flake8
```

Comment:
Is it necessary to write in newline?

Reply:
It should not be necesary if there is one line. I think there were multiple lines and then I removed one.